### PR TITLE
DSC-1310 adding ingress annotations

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

File: helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-welcome-to-prison-cert
     path: /
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```
Comment (andrewrlee) [inline helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml:17]: this needs to be inside an annotations section

Comment (michalnawrockidj) [reply helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml]: Upgraded to 1.1.2 version

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**